apps/docs/src/examples/inputs/PhoneInput.tsx

```diff
@@ -0,0 +1,91 @@
+import { useRef, useState } from "react";
+import { set } from "zod";
+import {
+  HvInput,
+  HvOption,
+  HvSelect,
+  HvTypography,
+} from "@hitachivantara/uikit-react-core";
+
+const formatPhoneNumber = (value: string) => {
+  const digits = value.replace(/\D/g, "");
+
+  // Apply formatting pattern: "XX XXX XX XX"
+  return digits
+    .slice(0, 9)
+    .replace(/(\d{2})(\d{3})?(\d{2})?(\d{2})?/, (match, p1, p2, p3, p4) => {
+      return [p1, p2, p3, p4].filter(Boolean).join(" ");
+    });
+};
+
+const countries = [
+  {
+    label: "India",
+    code: "+91",
+    flag: "🇮🇳",
+    format: formatPhoneNumber,
+  },
+  {
+    label: "Portugal",
+    code: "+351",
+    flag: "🇵🇹",
+    format: formatPhoneNumber,
+  },
+  {
+    label: "United Kingdom",
+    code: "+44",
+    flag: "🇬🇧",
+    format: formatPhoneNumber,
+  },
+  {
+    label: "United States",
+    code: "+1",
+    flag: "🇺🇸",
+    format: formatPhoneNumber,
+  },
+];
+
+export default function Demo() {
+  const [selectedCountry, setSelectedCountry] = useState("Portugal");
+  const [formatttedPhoneNumber, setFormattedPhoneNumber] = useState("");
+
+  return (
+    <div className="w-300px">
+      <HvInput
```

Comment:
let's avoid useless `div`s
```suggestion
      <HvInput
        className="w-300px"
```